javatests/arcs/core/entity/EntityBaseTest.kt

```diff
@@ -553,6 +553,8 @@ class EntityBaseTest {
       double = 16.0
       bigInt = BigInt.valueOf("6666666666666666666666666666666")
       bool = true
+      nullableBool = true
+      nullBool = null
```

Comment:
I see now.. Perhaps just use `nullableBool1` and `nullableBool2`? Or `nullableBool` and `nullableDouble`?

Reply:
Right, I had thought that the name 'nullBool' would point out that it was always intended to be null. I'll use your suggests though (for better coverage of the data types)